packages/terra-dialog/CHANGELOG.md

```diff
@@ -3,10 +3,12 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Updated `--terra-dialog-header-close-icon-margin-top` variable to only apply in Microsoft browsers
```

Comment:
Let's update this to say something along the lines of "Removed `--terra-dialog-header-close-icon-margin-top` variable, added `--terra-dialog-header-close-icon-ms-margin-top` variable to only apply top styling in Microsoft browsers.

Reply:
Updated in d761f5e